pages/mesosphere/dcos/2.2/installing/installation-faq/index.md

```diff
@@ -75,4 +75,4 @@ The IAM database is restored from the backup file and the cluster is operational
 
 ## Q. How do I back up ZooKeeper?
 
-To back up ZooKeeper, follow the guide to back up and restore the [DC/OS ZooKeeper State](/mesosphere/dcos/2.2/administering-clusters/backup-and-restore/backup-restore-cli/#zookeeper).
+To back up ZooKeeper, follow the guide to back up and restore the [DC/OS ZooKeeper State](/mesosphere/dcos/2.1/administering-clusters/backup-and-restore/backup-restore-cli/#zookeeper).
```

Comment:
ditto.

Reply:
well we can copy it to other versions aswell... I was thinking if we drop out the universal installer 0.2 docs in future releases and just refer back to the one when it was introduced...

@cmastr any thought about this

Reply:
ah guess this is a copy paste issue...